calcrule_social_protection/strategies/benefit_package_base_strategy.py

```diff
@@ -22,13 +23,25 @@ def calculate(cls, calculation, payment_plan, **kwargs):
         audit_user_id, start_date, end_date = \
             calculation.get_payment_cycle_parameters(**kwargs)
         user = User.objects.filter(i_user__id=audit_user_id).first()
+
+        #  ticket - at this stage use fixed amount, also skip ceiling part
```

Comment:
Please change that line into "#  TODO - ticket - at this stage use fixed amount, also skip ceiling part"

Reply:
done